metricbeat/module/system/process/process.go

```diff
@@ -0,0 +1,114 @@
+/*
+
+An example event looks as following:
+
+
+    {
+      "@timestamp": "2016-04-26T19:24:19.108Z",
+      "beat": {
+        "hostname": "ruflin",
+        "name": "ruflin"
+      },
+      "metricset": "process",
+      "module": "system",
+      "rtt": 20982,
+      "system-process": {
+        "processes": [
+          {
```

Comment:
The reason I added the additional "processes" array is that this will allow us to store additional data in the metricset if needed without changing the structure.